src/common/startup_check.py

```diff
@@ -352,6 +405,20 @@ async def _check_validators_type() -> None:
             )
 
 
+async def _check_reward_splitter_wallet() -> None:
+    chain_state = await get_chain_finalized_head()
+    reward_splitters = await graph_get_reward_splitters(
+        vaults=settings.vaults, block_number=chain_state.block_number
+    )
+    for reward_splitter in reward_splitters:
+        rewards_splitter_contract = RewardSplitterContract(reward_splitter.address)
+        if rewards_splitter_contract != hot_wallet.account.address:
+            raise RuntimeError(
+                f'Reward splitter claimer for vault {reward_splitter.vault} '
+                f'must equal to hot wallet address'
+            )
```

Comment:
As we gonna include wallet in the reward splitter query filter, we don't need to perform this check